spec/models/distribution_spec.rb

```diff
@@ -412,6 +417,8 @@ def create_priority_distributed_case(distribution, case_id, ready_at)
         let!(:legacy_nonpriority_cases) { [] }
         let!(:same_judge_nonpriority_hearings) { [] }
         let!(:other_judge_hearings) { [] }
+        # Proportion for hearings and evidence submission dockets
+        let(:other_dockets_proportion) { (1 - DocketCoordinator::MAXIMUM_DIRECT_REVIEW_PROPORTION) / 2 }
```

Comment:
Seem like this should account for legacy cases too:
```suggestion
        let(:other_dockets_proportion) { (1 - DocketCoordinator::MINIMUM_LEGACY_PROPORTION - DocketCoordinator::MAXIMUM_DIRECT_REVIEW_PROPORTION) / 2 }
```

Reply:
Nevermind, the test scenario is `when there are zero legacy cases eligible`.